packages/react-ui-components/src/MultiSelectBox/multiSelectBox.js

```diff
@@ -46,6 +46,14 @@ class MultiSelectBox extends PureComponent {
          */
         onValuesChange: PropTypes.func.isRequired,
 
+        /**
+         * This prop gets called when an option was selected,
+         * the inspector is not in dirty status and
+         * the NodeCreationDialog is not open.
+         * It handles the linking of the references editor.
+         */
```

Comment:
```suggestion
        /**
         * This prop gets called when one of the selected items gets clicked.
         */
```

This is for consistency.